src/engine/AbsoluteValueConstraint.cpp

```diff
@@ -329,6 +339,21 @@ List<PiecewiseLinearCaseSplit> AbsoluteValueConstraint::getCaseSplits() const
     return splits;
 }
 
+List<PhaseStatus> AbsoluteValueConstraint::getAllCases() const
+{
+    return { ABS_PHASE_NEGATIVE, ABS_PHASE_POSITIVE};
```

Comment:
PhaseStatus is a shared ENUM between all ConstraintClasses. I can add the constant with some generic name - BIPHASIC_PIECEWISE_LINEAR_CONSTRAINT. Although, it would make sense to have it as standalone constant to avoid mixing up status and number of cases. And it would be only constant of its kind, since all constraints so far are either fixed biphasic or have dynamic number of cases. What do you think?